commands/pull_request.go

```diff
@@ -202,6 +208,17 @@ func pullRequest(cmd *Command, args *Args) {
 		}
 	}
 
+	mm, _ := git.Config("hub.maintainerModify")
```

Comment:
Could you skip adding a global configuration option for now? A more generic way of defining default flags for hub commands globally is on the roadmap, and it's likely that it wouldn't be stored in `.gitconfig`.